yt/data_objects/static_output.py

```diff
@@ -116,11 +116,7 @@ def __get__(self, instance, owner):
             pass
         if self.display_array:
             try:
-                setattr(
-                    ret,
-                    "_ipython_display_",
-                    functools.partial(_wrap_display_ytarray, ret),
-                )
+                ret._ipython_display_ = functools.partial(_wrap_display_ytarray, ret)
```

Comment:
I think I wrote this code, and I think I had a reason for using `setattr` but for the life of me I can't remember, so 🤷

Reply:
I'll take the blame it this ever breaks :)